lib/_tls_wrap.js

```diff
@@ -881,6 +889,8 @@ exports.connect = function(/* [port, host], options, cb */) {
     options.singleUse = true;
 
   assert(typeof options.checkServerIdentity === 'function');
+  assert(typeof options.minDHSize === 'number');
+  assert(options.minDHSize > 0);
```

Comment:
Added assert messages. I also add a new commit to add an assert message for  options.checkServerIdentity as in https://github.com/shigeki/io.js/commit/e4f02bdce31c14dccb128a9adbf4ec92fe0296c5. I hope you like it.